### ISSUE TITLE
don't see the main or run .py

### ISSUE THREAD

**Issue by MNCTTY** [timxzz/GANs#2]
Hi! Thank you for sharing your pretrained model
But in FACE repository I see a lot of notebooks and really don't understand to which of them I must load the model. Can you help, please?

By the way, the server is offline again. Just in case, if you don't know.

**Comment by MNCTTY** [comment]
Ah, I've found your repo with backend
I ran 2 docker commands and the log is 
![image](https://user-images.githubusercontent.com/37251686/59948483-d90cec80-9478-11e9-82cd-68ceb40bc3a5.png)
But there is nothing at all by this address
![image](https://user-images.githubusercontent.com/37251686/59948538-ff328c80-9478-11e9-92cf-863fc29ba269.png)
It says that browser can't connect to this ip address.
What should I do else?

**Comment by timxzz** [comment]
@MNCTTY Yes, it should be like this as it is a API backend, it doesn't have website to get to. 

However, with the pretrained model, you should be able to run the Notebook stated in the README file: 
 [https://github.com/timxzz/GANs/blob/master/Face/Face_StarGAN_v5.ipynb](https://github.com/timxzz/GANs/blob/master/Face/Face_StarGAN_v5.ipynb)
